staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -298,7 +298,8 @@ type VirtualMachineInstanceStatus struct {
 	// MigratedVolumes lists the source and destination volumes during the volume migration
 	// +listType=atomic
 	// +optional
-	MigratedVolumes []StorageMigratedVolumeInfo `json:"migratedVolumes,omitempty"`
+	MigratedVolumes          []StorageMigratedVolumeInfo `json:"migratedVolumes,omitempty"`
+	VolumeMigrationSucceeded *bool                       `json:"volumeMigrationSucceeded,omitempty"`
```

Comment:
What are the pros/cons of adding these new bool fields instead of conditions?

Reply:
We can also add a condition instead of a bool

Reply:
@mhenriks I was trying to add the conditions here, however, they are usually part of a `Status`. Doesn't sound here a bit odd?